db/seeds.rb

```diff
@@ -35,25 +35,26 @@ def create_statuses
   end
 
   def create_items
-    Item.create!(title: "Sean's Footlong", description: "A hotdog with ketchup", price: 200)
-    Item.create!(title: "Yellow Dog", description: "A hotdog with mustard", price: 250)
+    Item.create!(title: "Turing Assessment Week", description: "A miserable dog seasoned with the tears of students", price: 200, image: File.open('app/assets/images/sad_dog.jpg'))
+    Item.create!(title: "Yellow Dog", description: "A hotdog with mustard", price: 250, image: File.open('app/assets/images/hotdog_costume.jpg'))
```

Comment:
Line is too long. [145/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.